[#652] Remove tilt infrastructure (#657)
boozallen/aissemble: extensions/extensions-helm/aissemble-localstack-chart/README.md

Baseline Helm chart for packaging and deploying localstack. The chart is built and managed during the normal Maven build
lifecycle and placed in the **target/helm/repo** directory.

> For aiSSEMBLE-based projects, note that this chart is intended only for local development work, such as when deploying via Helmfile to a local cluster.

Reviewer: A:
```suggestion
For aiSSEMBLE-based projects, note that this chart is intended only for local development to mimic AWS services.
```

Author: changed
